light-client-core/src/main/scala/ch/epfl/ognjanovic/stevan/tendermint/verified/blockchain/BlockchainSystem.scala

```diff
@@ -2,14 +2,16 @@ package ch.epfl.ognjanovic.stevan.tendermint.verified.blockchain
 
 import ch.epfl.ognjanovic.stevan.tendermint.verified.blockchain.BlockchainStates._
 import ch.epfl.ognjanovic.stevan.tendermint.verified.blockchain.SystemSteps.SystemStep
-import ch.epfl.ognjanovic.stevan.tendermint.verified.types.Chain.Genesis
 import ch.epfl.ognjanovic.stevan.tendermint.verified.types._
+import ch.epfl.ognjanovic.stevan.tendermint.verified.types.Chain.Genesis
 import stainless.annotation._
 import stainless.lang._
+import stainless.proof.check
+import utils.ListSet
 
 object BlockchainSystem {
 
-  @ghost
+  @ghost @extern
```

Comment:
Why is this extern now?

Reply:
It timeouts now. This is next on the cleanup list